libirods_rule_engine_plugin-indexing.cpp

```diff
@@ -201,6 +201,14 @@ namespace {
                     object_path,
                     _rei->rsComm->clientUser.userName,
                     source_resource);
+
+                const char* metadata_included = getValByKey(&obj_inp->condInput, METADATA_INCLUDED_KW);
+                if (metadata_included)
+                {
+                    idx.schedule_metadata_indexing_event(
+                        object_path,
+                        _rei->rsComm->clientUser.userName, "a", "v", "u"); // the actual AVU values do not matter since commit 277933912f
```

Comment:
good catch

Reply:
not sure about putting SHAs in the code itself.    consider how to explain what you want to convey to future readers without a reference into the timemachine.

Reply:
Like maybe : "since move to NIEHS metadata schema" ?

Reply:
You could provide an overload that wraps the line being discussed, but it takes no AVUs.

Same function name with fewer parameters.

Reply:
Still a pretty minor concern compared to the overall complexity of the plugin.  I could add such a wrapper or overload, but I don't know if it would make things clearer.

Do we want to make that its own issue perhaps?  Possibly even connected with an eventual move of this code to the policy-composed version of indexing?

Reply:
Create an issue for overloading/refactoring the function.

Let's not worry about policy-composed just yet. We can handle that later/separately.

Reply:
done.